src/pages/explore/dex/_components/ExploreDTokensSection.tsx

```diff
@@ -0,0 +1,210 @@
+    QQQ: t("exploreDTokensSection.tokenNames.QQQ"),
+  };
+
+  useEffect(() => {
+    api.poolpairs.listDexPrices("USDT").then((res) => {
+      setTokens(
+        tokenNameMap,
+        res,
+        setCryptoTokens,
+        setDstocksTokens,
+        setEtfsTokens,
+        setSelectedTabTokens
+      );
+    });
+  }, [api.poolpairs]);
+
+  useEffect(() => {
+    switch (tokenSectionChoice) {
+      case TokenLabels.ETFS:
+        setSelectedTabTokens(etfsTokens);
+        break;
+      case TokenLabels.DSTOCKS:
+        setSelectedTabTokens(dstocksTokens);
+        break;
+      case TokenLabels.CRYPTO:
+      default:
+        setSelectedTabTokens(cryptoTokens);
+    }
+  }, [tokenSectionChoice]);
+
+  return (
+    <Container className="mb-[240px]">
+      <div className="flex flex-col md:items-center items-start">
+        <SectionTitle text={t("exploreDTokensSection.title")} />
+        <div
+          className={classNames(
+            "lg:text-[60px] lg:leading-[72px] mt-5 lg:tracking-normal",
+            "text-[40px] leading-[44px] tracking-[-0.02em]"
+          )}
+        >
+          {t("exploreDTokensSection.subtitle")}
+        </div>
+
+        <ExploreDTokensTabs
+          tokenSectionChoice={tokenSectionChoice}
+          setTokenSectionChoice={setTokenSectionChoice}
+        />
+        <ExploreDTokensGrid selectionTokens={selectedTabTokens} />
+
+        <SecondaryButton
+          className="place-self-center mt-16 lg:py-4 py-3 md:px-[56px] px-[43px] w-fit lg:text-base text-sm lg::font-bold font-semibold"
+          text={t("exploreDTokensSection.secondaryButtonText")}
+          hasDiagonalArrow
+        />
+
+        <div className="text-dark-700 md:text-base text-sm place-self-center mt-4 text-center font-desc">
+          {t("exploreDTokensSection.pricesDesc")}
+        </div>
+      </div>
+    </Container>
+  );
+}
+
+const crypto = ["DUSD", "BTC", "ETH", "USDC", "EUROC", "USDT", "LTC", "BCH"];
+const dStocks = [
+  "TSLA",
+  "BABA",
+  "GOOGL",
+  "AAPL",
+  "AMZN",
+  "NVDA",
+  "MSFT",
+  "PYPL",
+];
+const etfs = ["ARKK", "SPY", "GLD", "VNQ", "VOO", "SLV", "URTH", "QQQ"];
+
+function reorderObject(obj, order) {
+  const newObj = {};
+  order.forEach((key) => {
+    if (Object.hasOwn(obj, key)) {
+      newObj[key] = obj[key];
+    }
+  });
+  return newObj;
+}
+
+function setTokens(
+  tokenNameMap: { [symbol: string]: string },
+  allUncleanTokens: DexPricesResult,
+  setCryptoTokens: Dispatch<
+    SetStateAction<{
+      [symbol: string]: DexPrice;
+    }>
+  >,
+  setDstocksTokens: Dispatch<
+    SetStateAction<{
+      [symbol: string]: DexPrice;
+    }>
+  >,
+  setEtfsTokens: Dispatch<
+    SetStateAction<{
+      [symbol: string]: DexPrice;
+    }>
+  >,
+  setSelectedTabTokens: Dispatch<
+    SetStateAction<{
+      [symbol: string]: DexPrice;
+    }>
+  >
+) {
+  // eslint-disable-next-line no-param-reassign
+  allUncleanTokens.dexPrices.USDT = {
```

Comment:
Curious to know why we need to hardcode this~

Reply:
cause we are using USDT prices if we call from ocean cause the denomination is USDT the dexprices wouldnt include USDT
https://ocean.defichain.com/v0/mainnet/poolpairs/dexprices?denomination=USDT